src/plugins/data_importer/server/routes/cat_indices.ts

```diff
@@ -14,17 +14,17 @@ export function catIndicesRoute(
   config: TypeOf<typeof configSchema>,
   dataSourceEnabled: boolean
 ) {
-  router.post(
+  router.get(
     {
       path: '/api/data_importer/_cat_indices',
-      validate: {
-        query: schema.object({
-          dataSource: schema.maybe(schema.string()),
-        }),
-      },
+      validate: false,
```

Comment:
why aren't we validating here? shouldn't we enforce `dataSourceId` if it's provided as a query param?